certified-connectors/Docurain/apiDefinition.swagger.json

```diff
@@ -0,0 +1,248 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Docurain",
+    "description": "DocurainAPI Communication Connector. Docurain API allows you to create your own documents using template files registered in Docurain and data from PowerAppsn.",
+    "contact": {
+      "name": "Docurain support",
+      "url": "https://api.docurain.jp/console/inquiry",
+      "email": "cs@docurain.jp"
+    }
+  },
+  "host": "powerapps.docurain.jp",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/apiw/{outType}/{templateName}": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "title": "body",
+              "type": "string",
+              "format": "binary",
+              "description": "If the process is successful, the binary data of the generated form is returned. If the process fails, JSON with the details of the error is returned."
+            },
+            "headers": {
+              "Content-Type": {
+                "description": "Content-Type",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "summary": "Render",
```

Comment:
@geckoR42 It looks more like a operationId rather than a summary. As the values of summary and descriptions are visible to public, the values should describe concisely the main purpose of the operation in a sentence.

Reply:
@sriyen-msft Fixed "summary"